libkbfs/kbfs_ops_concur_test.go

```diff
@@ -1900,3 +1900,160 @@ func TestKBFSOpsCancelGetFavorites(t *testing.T) {
 	}
 	testRPCWithCanceledContext(t, serverConn, f)
 }
+
+type stallingNodeCache struct {
+	NodeCache
+
+	doStall <-chan struct{}
+	unstall <-chan struct{}
+	blocked chan<- struct{}
+	paths   chan<- struct{}
+}
+
+func (snc *stallingNodeCache) UpdatePointer(
+	oldRef BlockRef, newPtr BlockPointer) {
+	select {
+	case <-snc.doStall:
+		snc.blocked <- struct{}{}
```

Comment:
I think you can remove this line -- see comment below. Probably best anyway, to avoid 'crossing the streams', since snc.blocked is used by PathFromNode.